src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -6018,8 +6444,15 @@ public override IEnumerable<IOperation> Children
         {
             get
             {
-                yield return InitializedMember;
-                yield return Initializer;
+                IOperation current;
+                if ((current = Initializer) != null)
+                {
+                    yield return current;
+                }
+                if ((current = InitializedMember) != null)
```

Comment:
>if ((current = InitializedMember) != null) [](start = 16, length = 42)

I think the initialized member should be returned first.

Reply:
done